hashintel/hash
# Update boolean chip design on entity editor property table

## 🌟 What is the purpose of this PR?

This PR updated the design of boolean editor on entity editor to have better UX and visual clarity.

## 🔗 Related links

- [Asana task](https://app.asana.com/0/1203363157432084/1203556537681020/f) _(internal)_
- [Design](https://www.figma.com/file/gydVGka9FjNEg9E2STwhi2/HASH-App?node-id=6295%3A251885&t=yqgfuFMwSpg2gE9t-4) _(internal)_


## ❓ How to test this?



1.  Checkout the branch / view the deployment
1.  Open entity editor
2. Confirm boolean values works as expected

## 📹 Demo

https://user-images.githubusercontent.com/39553853/209373474-8f83a15e-3d06-41f4-9f7b-bca0a9325c4f.mov
